sidebars.ts

```diff
@@ -37,20 +32,29 @@ const sidebars: SidebarsConfig = {
           id: "frontend/index",
```

Comment:
kun je de folder renamen naar `front-end`? dan hebben we het overal hetzelfde (met dash)